docs/repositories/commits.md

```diff
@@ -100,6 +100,17 @@ The following are example situations that can lead to potential issues:
 
     This happens when Codacy reports potential issues in files that weren't changed in your pull request. [Read more about this GitHub feature](https://developer.github.com/changes/2019-09-06-more-check-annotations-shown-in-files-changed-tab/).
 
+### False positive issues
+
+!!! info
+    This feature is available only on the Business tier or during a trial.
```

Comment:
i'd remove mentions to trial so users dont expect this to be on by default during a trial
```suggestion
    This feature is available only on the Business tier. If you are a Codacy Pro customer, contact us to access a short trial.
```